org.garudalinux.firedragon.appdata.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.garudalinux.firedragon</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MPL-2.0 AND LicenseRef-proprietary=https://raw.githubusercontent.com/Floorp-Projects/Floorp-private-components/main/LICENSE</project_license>
+  <name>FireDragon</name>
+  <summary><![CDATA[Floorp fork build using custom branding & settings]]></summary>
```

Comment:
here too. I shouldn't have to point it all.

wherever you copied that from it's wrong.

Reply:
Fixed in dfa468ff8493ceb554dd637c6212885fde5c57d4.